maintainer/change_release.sh

```diff
@@ -73,14 +73,13 @@ echo "Using sed = $SED"
 
 echo "Setting RELEASE/__version__ in MDAnalysis to $RELEASE"
 
-git grep -E -l 'RELEASE.*[0-9]+\.[0-9]+\.[0-9]+(\.[0-9]+)?(-dev*[0-9]*|-rc*[0-9]*)?' $FILES  \
-   | xargs -I FILE $SED
-   '/RELEASE/s/[0-9]+\.[0-9]+\.[0-9]+(\.[0-9]+)?(-dev*[0-9]*|-rc*[0-9]*)?/'${RELEASE}'/' -i.bak FILE
+git grep -E -l 'RELEASE.*[0-9]+\.[0-9]+\.[0-9]+(\.[0-9]+)?(-dev[0-9]*|-rc[0-9]*)?' $FILES  \
+   | xargs -I FILE $SED '/RELEASE/s/[0-9]+\.[0-9]+\.[0-9]+(\.[0-9]+)?(-dev[0-9]*|-rc*[0-9]*)?/'${RELEASE}'/' -i.bak FILE
 
-git grep -E -l '__version__=.*[0-9]+\.[0-9]+\.[0-9]+(\.[0-9]+)?(-dev*[0-9]*|-rc*[0-9]*)?'  $FILES \
-   | xargs -I FILE $SED '/__version__/s/[0-9]+\.[0-9]+\.[0-9]+(\.[0-9]+)?(-dev*[0-9]*|-rc*[0-9]*)?/'${RELEASE}'/' -i.bak FILE
+git grep -E -l '__version__.=.*[0-9]+\.[0-9]+\.[0-9]+(\.[0-9]+)?(-dev[0-9]*|-rc[0-9]*)?'  $FILES \
```

Comment:
Good catch, however, better use
```
__version__ *=
```
to allow any amount of space... maybe `\s*` works but I can't remember if regex has a special "white space" pattern.